crates/kubelet/src/kubelet.rs

```diff
@@ -37,7 +37,7 @@ pub struct NotImplementedError;
 ///
 /// This is specified by Kubernetes itself.
 #[derive(Clone, Debug, serde::Serialize)]
-pub enum Phase {
+pub(crate) enum Phase {
```

Comment:
would it be useful for providers to inspect the pod's phase?

Reply:
Based on how I have it implemented now, there should be no need to check the phase outside of the crate